src/dg/dg.cpp

```diff
@@ -434,6 +434,54 @@ void DGBaseState<dim,nstate,real,MeshType>::update_model_variables()
     pde_model_rad_fad->cellwise_volume.update_ghost_values();
 }
 
+template <int dim, int nstate, typename real, typename MeshType>
```

Comment:
Since these values are computed in `FlowSolver` through a `FlowSolverCaseBase` object and need to be assigned to the `ModelBase` object held in DG, I chose to implement a setter function in DG that we can call from within `FlowSolver` which has ; this is a different scenario than calling the "update_model_variables()" function since that function cannot compute quantities that are related to a specific physics or flow solver case